estate/models/estate_property.py

```diff
@@ -0,0 +1,27 @@
+from odoo import fields, models
+from datetime import datetime, timedelta
+
+class Property_Plan(models.Model):
+    _name = "estate_property"
+    _description = "Test Model"
+    name = fields.Char(string="Title",required=True)
+    description = fields.Text(string="Description")
+    postcode = fields.Char(string="Postcode")
+    date_availability = fields.Date(string="Date Availability", copy=False, default=lambda self: (datetime.today() + timedelta(days=90)).date())
+    expected_price = fields.Float(string="Expected Price",required=True)
+    selling_price = fields.Float(string="Selling Price",readonly=True,copy=False)
```

Comment:
```suggestion
    expected_price = fields.Float(string="Expected Price", required=True)
    selling_price = fields.Float(string="Selling Price", readonly=True, copy=False)
```

Please ensure the entire module follows coding guidelines by adding spaces wherever necessary.
 Also, could you explain the purpose of "required," "readonly," and "copy" parameter in this context?

Reply:
I have added the necessary spaces and blank lines for follow coding guidelines.
required parameter means that field is important to fill out which is mandatory.
readonly parameter is for for read only we cannot set the value.
copy parameter is set to false by which if we make duplicate property the selling price will not be copied.